### PR TITLE
feat(route): 充电头网

### PR DESCRIPTION
**Pull request by Geraldxm**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/chongdiantou
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [x] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Review by TonyRL — COMMENTED** [review]
Please use [WordPress REST API](https://developer.wordpress.org/rest-api/) instead.
The API endpoints are available at `/wp-json/wp/v2`.

For the latest post, you can use the endpoint of `/wp-json/wp/v2/posts`

If you want categories and tags for each post in the posts API, you can add the `_embed` search parameter. To adjust the no. of articles returned to the posts API, you can pass the `per_page` search parameter.

File: lib/routes/chongdiantou/index.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import { namespace } from './namespace';
+import ofetch from '@/utils/ofetch';
+import cache from '@/utils/cache';
+import logger from '@/utils/logger';
+
+async function getPosts() {
+    const cacheKey = 'chongdiantou_posts';
+    try {
+        // 尝试从缓存中获取数据，如果没有缓存则从 API 获取
+        const data = await cache.tryGet(cacheKey, async () => {
```
Comment (TonyRL) [inline lib/routes/chongdiantou/index.ts:11]: You don't need to cache that.

File: lib/routes/chongdiantou/index.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+            const response = await ofetch('https://www.chongdiantou.com/wp-json/wp/v2/posts?_embed&per_page=10', {
+                headers: {
+                    method: 'GET',
+                },
+            });
+            return response.map((post) => ({
+                title: post.title.rendered,
+                link: post.link,
+                pubDate: new Date(post.date),
```
Comment (TonyRL) [inline lib/routes/chongdiantou/index.ts:20]: Use date_gmt instead.

File: lib/routes/chongdiantou/index.ts (hunk 1 above)
Comment (Geraldxm) [reply lib/routes/chongdiantou/index.ts]: Why is caching not necessary here?

File: lib/routes/chongdiantou/index.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+export const route: Route = {
+    path: '/',
+    categories: namespace.categories,
+    example: '/chongdiantou',
+    radar: [
+        {
+            source: ['www.chongdiantou.com'],
+        },
+    ],
+    name: '充电头网',
```
Comment (TonyRL) [inline lib/routes/chongdiantou/index.ts:38]:
```suggestion
    name: '最新资讯',
```

**Pull request merged**